docs/dev.rst

```diff
@@ -197,9 +227,9 @@ Then, you should run the following commands::
     # Rebuild static assets
     openedx-assets build --env=dev
 
-To debug a local edx-platform repository, add a ``import ipdb; ipdb.set_trace()`` breakpoint anywhere in your code and run::
+After running all these commands, your edx-platform repository will be ready for local development. To debug a local edx-platform repository, you can then add a ``import ipdb; ipdb.set_trace()`` breakpoint anywhere in your code and run::
 
-    tutor dev runserver [--volume=...] lms
+    tutor dev runserver --mount=/path/to/edx-platform lms
```

Comment:
Hm, thinking forward to the `runserver` deprecation... without `runserver`, how would folks interact with pdb?

Do we need a new `attach` command?
```
tutor dev start -d --mount=/path/to/edx-platform lms
tutor dev attach lms
```